plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -228,6 +262,9 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
 
    std::string default_wasm_runtime_str= eosio::chain::wasm_interface::vm_type_string(eosio::chain::config::default_wasm_runtime);
 
+   // It is optimal that number of threads equals to the number of cores
+   uint16_t default_rocksdb_threads = std::thread::hardware_concurrency();
```

Comment:
This is not a good default. There are many other threads in play besides just rocksdb.

Reply:
We were discussing the same concern when put the hardware_concurrency here. We thought as this was configurable, it could be changed if it turns out to be bad. But we can just set the default to 1.

Reply:
Made 1